pom.xml

```diff
@@ -216,6 +219,11 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.sonarsource.scanner.maven</groupId>
```

Comment:
⛏️ Needs formatting.

Reply:
The format should be correct. Is something wrong with it?

Reply:
Needs leading whitespace.